src/platform/tigerlake/lib/clk.c

```diff
@@ -10,20 +10,24 @@
 #include <sof/lib/clk.h>
 
 const struct freq_table platform_cpu_freq[] = {
+	{ 38400000, 38400 },
 	{ 120000000, 120000 },
 	{ CLK_MAX_CPU_HZ, 400000 },
 };
 
 const uint32_t cpu_freq_enc[] = {
+	SHIM_CLKCTL_WOVCROSC | SHIM_CLKCTL_WOV_CRO_REQUEST |
+		SHIM_CLKCTL_HMCS_DIV2 | SHIM_CLKCTL_LMCS_DIV4,
 	SHIM_CLKCTL_RLROSCC | SHIM_CLKCTL_OCS_LP_RING |
 		SHIM_CLKCTL_HMCS_DIV2 | SHIM_CLKCTL_LMCS_DIV4,
 	SHIM_CLKCTL_RHROSCC | SHIM_CLKCTL_OCS_HP_RING |
 		SHIM_CLKCTL_HMCS_DIV2 | SHIM_CLKCTL_LMCS_DIV4,
 };
 
 const uint32_t cpu_freq_status_mask[] = {
+	SHIM_CLKCTL_WOV_CRO_REQUEST,
```

Comment:
better to redefine sth. like SHIM_CLKSTS_WOVROSCS to avoid confusion though it is actually same value.

Reply:
Sure, I will do that